AmazonChimeSDK/AmazonChimeSDK/internal/audio/DefaultAudioClientObserver.swift

```diff
@@ -197,6 +198,62 @@ class DefaultAudioClientObserver: NSObject, AudioClientDelegate {
         }
     }
 
+    public func transcriptEventsReceived(_ events: [Any]?) {
+        if events == nil {
```

Comment:
Does `guard let internalEvents = events` below include this nil check?

Reply:
Good point, removed nil check.